src/ui/LanguageStatusItems.ts

```diff
@@ -0,0 +1,75 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the VSCode Swift open source project
+//
+// Copyright (c) 2021 the VSCode Swift project authors
```

Comment:
The copyright should be updated for new files to point to 2022

Reply:
Damn keep missing that